docs/customization/querying-columns.md

```diff
@@ -0,0 +1,16 @@
+# Querying columns with SQL
+
+The model `int_all_columns` ([source](https://github.com/dbt-labs/dbt-project-evaluator/tree/main/models/marts/core/int_all_columns.sql)), created with the package, lists all the columns from all the dbt nodes (models, sources, seeds, snapshots, analyses)
+
+You can use this model to help with questions such as:
+
+- Are there columns with the same name in different nodes?
+- Are there columns that lack descriptions?
```

Comment:
This will only look at columns that have been manually added to the YAML files and not all the columns of every model. Could we re-word this sentence to mention it?